tests/status.py

```diff
@@ -72,6 +72,35 @@ def runTest(self):
         board.expect(prompt)
         self.result_message = 'ifconfig shows ip addresses: %s' % tmp
 
+class MacAddressCheck(rootfs_boot.RootFSBootTest):
+    '''Checked MAC addresses for all interfaces.'''
+    def runTest(self):
+        board.sendline('\nhexdump -n 6 -v -e \'/1 "%02X"\' /proc/device-tree/uccp@18480000/mac-address0')
+        board.expect('0019F5890AFB.*')
```

Comment:
This test has to work on all boards we plug into boardfarm, this is not portable enough. No board has come off production that doesn't fit into 0019F589.* so we should probably bump up the mask to that.